vercel.prod.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "jsPerf",
+    "version": 2,
+    "public": true,
+    "alias": ["jsperf.com"],
+    "env": {
+      "NODE_ENV": "production",
```

Comment:
You don't need another config file here if you're only changing env vars.

Instead, use the project settings to configure different values for Preview vs Production.

https://vercel.com/blog/environment-variables-ui

Furthermore, if you are only using `NODE_ENV`, for your serverless functions, then you shouldn't need to assign this at all.